action.yml

```diff
@@ -28,9 +30,9 @@ runs:
     - id: determine_token
       run: |
         if [ -n "$INPUT_TOKEN" ]; then
-          token=$INPUT_TOKEN
+          token="$INPUT_TOKEN"
         else
-          token=$DEFAULT_TOKEN
+          token="$DEFAULT_TOKEN"
         fi
         echo "TOKEN=$token" >> "$GITHUB_OUTPUT"
```

Comment:
Thank you for cleaning up some of the other code!

This is one of those oft forgotten and annoying shell script isms that https://github.com/koalaman/shellcheck would suss out but always awkward to ask a contributor to clean up our messes.

Reply:
No worries, easy to spot once you're at it but otherwise easy to miss!